src/wp-content/themes/twentyfifteen/functions.php

```diff
@@ -432,7 +415,8 @@ function twentyfifteen_javascript_detection() {
  */
 function twentyfifteen_scripts() {
 	// Add custom fonts, used in the main stylesheet.
-	wp_enqueue_style( 'twentyfifteen-fonts', twentyfifteen_fonts_url(), array(), null );
+	$font_version = ( 0 === strpos( (string) twentyfifteen_fonts_url(), get_template_directory_uri() . '/' ) ) ? '20230328' : null;
```

Comment:
Can we use `str_starts_with` here?

Reply:
Unfortunately, no. These themes should work with older versions of both PHP and WordPress.